CMSgov/onemac
# Oy2 22186 - remove left hand nav

Story: https://qmacbis.atlassian.net/browse/OY2-22186
Endpoint: See github-actions bot comment

### Details

Remove left hand navigation. Move additional details to bottom. Update TE submission landing page to package list.

### Changes

- removed all left hand nav functionality
- updated detailview to always include additional info section and removed conditional logic on detail section so that it never displays additional info section
- update TE form landing page

### Implementation Notes

- I thought about commenting it all out, but it was just too much for me to comfortably leave in. Source control will retain what was done if we ever put back a left hand nav.

### Test Plan

1. Login as any user with view privileges (such as state submitter)
2. Navigate to package details page for each package type and verify additonal information section always appears
3. When no additional information has been submitted then verify the default text stating as such.
